pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -128,12 +143,12 @@ pub mod pallet {
 		/// The type used for the curve parameters.
 		type CurveParameterType: Parameter
 			+ Member
-			+ FixedPointNumber<Inner = u128>
+			+ FixedSigned
 			+ TypeInfo
 			+ MaxEncodedLen
 			+ Zero
-			+ CheckedAdd
-			+ CheckedSub;
+			+ PartialOrd<Precision>
```

Comment:
Yes true, but the trait bounds are: 

```
pub fn exp<S, D>(mut operand: S) -> Result<D, ()>
where
    S: FixedSigned + PartialOrd<ConstType>,
    D: FixedSigned + PartialOrd<ConstType> + From<S> + From<ConstType>,
```
I use for S and D the same type. It would work out for D, but then I get compile issues for S.

Reply:
Ok, makes sense.